cfme/tests/containers/test_containers_default_views.py

```diff
@@ -43,9 +50,11 @@ def test_containers_providers_default_view(button_group, view):
               to Grid/Tile/List view
             * Goes to Compute --> Containers --> Providers and verifies the selected view
         """
-    sel.force_navigate('my_settings_default_views')
+    navigate_to(Server, 'MySettings')
```

Comment:
Yep, thanks for being patient. I've got an approval for that PR, might be able to get it merged today or tomorrow.